dspace-api/pom.xml

```diff
@@ -323,6 +325,11 @@
             <artifactId>apache-jena-libs</artifactId>
             <type>pom</type>
         </dependency>
+        <dependency>
+            <groupId>commons-cli</groupId>
+            <artifactId>commons-cli</artifactId>
+            <version>1.4</version>
```

Comment:
Should this change be in the Parent POM (or maybe just removed)?  It looks like in the Parent POM (`[src]/pom.xml`) we already have commons-cli version 1.3.1 right now...So, it's unclear why we have a different version specified here.

Reply:
Thank you.  It should be declared here (since this project uses it) and updated in the parent (since it is already managed there).  I'll do that.